src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -486,6 +486,39 @@ public unsafe bool CursorGrabbed
             }
         }
 
+        /// <inheritdoc />
+        public int CurrentMonitorPpi
+        {
+            get
+            {
+                int widthInMm;
+                int heightInMm;
+                int widthInPixels;
+                int heightInPixels;
+                float horizontalPpi;
+                float verticalPpi;
+
+                unsafe
+                {
+                    GraphicsLibraryFramework.Monitor* monitor =
+                        CurrentMonitor.ToUnsafePtr<GraphicsLibraryFramework.Monitor>();
+                    VideoMode* videoMode;
+
+                    GLFWProvider.GLFW.Value.GetMonitorPhysicalSize(monitor, out widthInMm, out heightInMm);
```

Comment:
Can `widthInMm` and/or `heightInMm` ever be zero? if so, the formula below will result in an `DivisionByZeroException`

Reply:
maybe if minimized.

Reply:
Could be avoided by multiplying by 0.0393700787 (1/25.4).

Reply:
That's also possible, but I'd write it out as 1/25.4f, makes it more obvious what it is, and the compiler will reduce it later anyway.

Reply:
Issue should be resolved in the latest commit.